lib/Coocook/Controller/Recipe.pm

```diff
@@ -107,6 +107,7 @@ sub edit : GET HEAD Chained('base') PathPart('') Args(0) RequiresCapability('vie
       and $c->stash(
         import_url => $c->uri_for_action( '/browse/recipe/import', [ $recipe->id, $recipe->url_name ] ) );
 
+    push @{ $c->stash->{js} }, '/js/unsavedChanges.js';
```

Comment:
I like your generic implementation with the `<form>` class. Do you think we should simply add this to our global `script.js` to utilize it on any page? :thinking: